### ISSUE TITLE
Armor Schreen

### ISSUE THREAD

**Issue by WiiZy500** [q13er/gary8#19]
Wearing draconic armor fucks with the chromatic lexicon screen

**Comment by q13er** [comment]
Yeah it seems to go opaque. I wonder if it is the shield GUI in the corner that is conflicting with it
